### DISCUSSION TITLE
Random "sort"

### DISCUSSION THREAD

**Discussion by wisp3rwind** [beetbox/beets#4443] [Ideas]
Just to write down the idea: While thinking about limiting results for #4362, and the usecase of `smartplaylist`s mentioned in https://github.com/beetbox/beets/pull/4190/files, it occurred to me that this combination of features might benefit from randomizing the track's order before applying the limit. That should be fairly easy to implement, via [SQLite `ORDER BY random()`](https://stackoverflow.com/a/1253576/3451198).

(Not going to implement this myself probably; I'm not using the plugin.)

**Comment by sampsyo** [comment]
Indeed, this does actually sound awesome.
